ts/src/index.ts

```diff
@@ -72,8 +72,12 @@ export async function initConfig(config: Config): Promise<ProfilerConfig> {
 
   let envSetConfig: Config = {};
   if (process.env.hasOwnProperty('GCLOUD_PROFILER_CONFIG')) {
+    // Type of process.env.GCLOUD_PROFILER_CONFIG is string | undefined
+    // but we know that it is defined.
+    // TODO: figure out how to avoid this cast.
     envSetConfig =
-        require(path.resolve(process.env.GCLOUD_PROFILER_CONFIG)) as Config;
+        require(path.resolve(process.env.GCLOUD_PROFILER_CONFIG as string)) as
```

Comment:
Try `process.env.GCLOUD_PROFILER_CONFIG!`, which coerces it to a non-null type. We should use it sparingly but in this case it's fine because we're already calling `hasOwnProperty` immediately beforehand